139_Client_Server/python_server.py

```diff
@@ -0,0 +1,63 @@
+import socket # Gives us #socket.socket() and socket.sock_stream()
+import threading # For multiple connections
+
+clients = []
+
+
+# Thread for each client
+def serve(client_connect, client_address, client_id):
+  global clients
+  try:
+    print(f"Serve {client_id}")
+    while(1):
+      client_connect.sendall(bytes('Write something!',encoding='utf8'))
```

Comment:
```suggestion
      client_connect.sendall(b'Write something!')
```
Use `.encode('utf-8')` to go from Unicode strings to bytes, `.decode('utf-8')` for the opposite, unless you have literals, in that case just use `'...'` for Unicode strings, and `b'...'` for bytes.